Data-Science/Kubeflow/MNIST-Digits-Recognition/mnist_katib_tf_kserve_example.ipynb

```diff
@@ -568,19 +575,18 @@
     "    name=name,\n",
     "    description=\"An end to end mnist example including hyperparameter tuning, train and inference\"\n",
     ")\n",
-    "def mnist_pipeline(name=name, namespace=namespace, training_steps=training_steps):\n",
+    "\n",
+    "def mnist_pipeline(name: str, namespace: str):\n",
     "\n",
     "    # Run the hyperparameter tuning with Katib.\n",
     "    katib_op = create_katib_experiment_task(name, namespace, training_steps)\n",
-    "\n",
-    "    volume_str = f\"model-volume-{uuid}\"\n",
     "    \n",
     "    # Create volume to train and serve the model.\n",
-    "    model_volume_op = dsl.VolumeOp(\n",
-    "        name=volume_str,\n",
-    "        resource_name=volume_str,\n",
+    "    model_volume_op = kfp.kubernetes.CreatePVC(\n",
+    "        pvc_name=volume_str,\n",
     "        size=\"1Gi\",\n",
-    "        modes=dsl.VOLUME_MODE_RWO\n",
+    "        storage_class_name=\"<storage_class_name>\",\n",
```

Comment:
We can use something like this instead of expecting user to know this and notice to change.
```
storage_class_name = !kubectl get pvc user-pvc -o jsonpath={.spec.storageClassName}
storage_class_name = storage_class_name[0]
```

Reply:
Updated the code with the above code snippet.